tests/ui/suggestions/crate-or-module-typo.stderr

```diff
@@ -41,7 +41,7 @@ LL -     bar: st::cell::Cell<bool>
 LL +     bar: cell::Cell<bool>
    |
 
-error[E0433]: failed to resolve: function `bar` is not a crate or module
+error[E0433]: cannot find `bar` in this scope
```

Comment:
Ah, I found I'd opened a separate PR already with some of these changes, and you had a comment about this change in particular

https://github.com/rust-lang/rust/pull/128086#discussion_r2111854198

I agree that this isn't an improvement, but the primary label does still mention the intended issue. I'll see over the weekend what I can do about this.